### DISCUSSION TITLE
DietPi-Software | Coder: VSCode in browser

### DISCUSSION THREAD

**Discussion by Gill-Bates** [MichaIng/DietPi#4940] [Software Request]
Hey there,

I would like to suggest Code Server for Dietpi. I have it successfully running:

https://github.com/cdr/code-server

![04C3E132-96A2-4EC1-A2F6-786C25A5C081](https://user-images.githubusercontent.com/25774289/140643469-3802e9b3-bbf7-4744-b1cd-5a9bb5781348.jpeg)

**Comment by Joulinar** [comment]
We are moving to GitHub discussion to collect software and image request in future. Therefore `issue` will be closed and reopen as `discussion.`

**Comment by MichaIng** [comment]
Many thanks for your request.

Currently the FOSS VSCodium variant is available. Is there a specific reason you prefer VSCode over VSCodium?

**Reply by Gill-Bates** [reply]
Just because it’s not available:

https://github.com/VSCodium/vscodium/issues/205

**Reply by MichaIng** [reply]
Ah I misunderstood the request. It's about "Coder", a web interface for VSCode moreless: https://coder.com/
Indeed a VSCodium variant would be nice.

**Comment by Gill-Bates** [comment]
During further tests it turned out that code server is only available for armhf (64 bit).

**Reply by MichaIng** [reply]
`armhf` == 32-bit: There are three Debian packages with every release: `amd64`, `arm64` and `armhf`, so all architectures covered (ARMv6 needs testing).
